app/models/competition.rb

```diff
@@ -321,6 +322,18 @@ def has_administrative_notes?
     end
   end
 
+  validate :auto_close_threshold_validations, if: -> { auto_close_threshold? }
```

Comment:
No, I literally meant...
```suggestion
  validate :auto_close_threshold_validations, if: :auto_close_threshold?
```

Reply:
For context, Ruby/Rails can use symbols to invoke methods, but these obviously only work for simple use-cases (i.e. if the check that you want to use is actually its own method). Opening lambdas with `-> { }` is really only worth it if you're introducing complex logic

Reply:
Ah of course! That's much simpler